app/javascript/mastodon/locales/fr.json

```diff
@@ -32,7 +32,7 @@
   "account.unmute": "Ne plus masquer",
   "account.unmute_notifications": "Réactiver les notifications de @{name}",
   "account.view_full_profile": "Afficher le profil complet",
-  "alert.unexpected.message": "Une erreur non-attendue s'est produite.",
+  "alert.unexpected.message": "Une erreur non-attendue s’est produite.",
```

Comment:
«Non attendue» without an hyphen which is used in common names but not for adjectives